README.md

```diff
@@ -343,6 +343,18 @@ Filename: test.txt
 * Note2 : Use `NON_ASCII_FONTS` to access all Non-ASCII fonts name list (new in `Version 4.4`)
 * Note3 : Use `ASCII_FONTS` to access all ASCII fonts name list (new in `Version 5.7`)
 
+### 4. lprint
```

Comment:
I believe that the `lprint` and `line` functions are not part of the `## ASCII text` section. We need to create a new section named `Line` to include both of these functions.